nat-lab/data/teliod/config.json

```diff
@@ -2,5 +2,6 @@
     "log_level": "trace",
     "log_file_path": "teliod_natlab.log",
     "authentication_token": "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
-    "app_user_uid": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee"
+    "app_user_uid": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee",
+    "interface_name": "teliod"
```

Comment:
We'll need to remember to add in the readme that for macOS this has to be `utun#`